### PR TITLE
chore: replace old mui add button with html button

### PR DESCRIPTION
**Pull request by jenniferarnesen**
Getting rid of old material-ui (0.20)

This is the green + button in the upper left of the dashboard. You can look at it on different browsers at this url:  https://deploy-preview-306--dhis2-dashboards.netlify.com

### CODE REVIEW

**Review by cooper-joe — APPROVED** [review]

File: src/widgets/AddDashboardButton.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import AddIcon from '@material-ui/icons/AddCircle';
+import { colors } from '../modules/colors';
+
+const AddDashboardButton = () => {
+    return (
```
Comment (amcgee) [inline src/widgets/AddDashboardButton.js]: Minor style issue but wouldn't this be cleaner without the functional body?  I.E. `const AddDashboardButton = () => (<React.Fragment>...</React.Fragment>)`

**Review by amcgee — CHANGES_REQUESTED** [review]
This apprears to work as a global `<style>` but doesn't actually use `styled-jsx` because CRA needs an additional babel plugin to support it.  I'll look into how that might be achieved, but it may be non-trivial.

File: src/widgets/AddDashboardButton.js (hunk 1 above)
Comment (jenniferarnesen) [reply src/widgets/AddDashboardButton.js]: Done

**Comment by jenniferarnesen** [comment]
@amcgee 
So I've gone through some iterations to arrive at the current solution:
- 1) moved css to a separate file and used css modules. This led to having to tweak the index file to control the injection of material-ui css (due to a specificity clash between my css and material-ui/icons css)
- 2) Thought the above was too complicated, so decided to use the material svg markup directly and get rid of @material-ui/icons for this component.
- 3) Removed the button, because why have a button inside an anchor. This resulted in getting to remove most of the css. Now the css file with just a single rule seems almost silly, but I do like using css modules rather than the inline styles.

Let me know what you think.

**Comment by amcgee** [comment]
@jenniferarnesen looking now!

**Review by amcgee — APPROVED** [review]
Looks great, and very clean!

One possible enhancement would be to extract the SVG into a separate file so that it's self-contained and could potentially be re-used.

**Pull request merged**